_data/companies.yml

```
@@ -586,6 +586,13 @@
   events: "?" 
   last_update: 2020-03-11 
  
+- name: Kentico 
```

```suggestion
- name: Kentico[[1]](https://www.kentico.com/blog/how-kentico-is-coping-with-covid-19) 
```

Let's add the source link here, please.